app/views/layouts/application.html.erb

```diff
@@ -4,9 +4,6 @@
   <%= render "head" %>
   <body
     class="
-      h-screen
-      font-sans text-gray-950 dark:text-gray-100
```

Comment:
I see you moved h-screen, but did you mean to drop the `text-gray-*` changes?

Reply:
I moved those styles into the `@layer base body` declaration as well. My thinking, on the fly, was that I would move all of the tailwind classes into the directive and keep the `nav-closed` class in the markup, seeing as it isn't a tw class—despite seeing the other `nav-closed `declarations in the `application.tailwind.css`.